config/initializers/audit_socket.rb

```diff
@@ -25,11 +25,27 @@
 #
 #    Formatter docs:
 #    https://ruby-doc.org/stdlib-2.5.1/libdoc/logger/rdoc/Logger.html#class-Logger-label-Format
-if path = Rails.application.config.try(:audit_socket)
+Audit.logger = if path = Rails.application.config.try(:audit_socket)
   # TODO: Add tests for this.  Our CI isn't verifying if we've broken anything.
-  Audit.logger = Audit::Log::SyslogAdapter.new(
+  Audit::Log::SyslogAdapter.new(
     Logger.new(UNIXSocket.open(path)).tap do |logger|
       logger.formatter = Logger::Formatter::RFC5424Formatter
     end
   )
+else
+  # NOTE: Use Syslog format for audit events so we can write e2e tests against
+  # what actually appears in the log.  We don't want this to affect our other
+  # logging because the ConjurFormatter produces more human-friendly logs.
+  #
+  # NOTE: We need to use the relative path "./log/development.log"
+  # because in the CI environment (see "ci/docker-compose.yml") the
+  # cucumber and conjur containers mount the log volume at different
+  # absolute paths ("/src" vs "/opt").  Otoh, in LogHelpers::LOG_LOCATION
+  # -- which this code is coupled to -- we can use the absolute
+  # path "/src" since that runs in the cucumber container.
+  Audit::Log::SyslogAdapter.new(
+    Logger.new("./log/development.log").tap do |logger|
```

Comment:
> What is the impact of two Logger instances (audit and the Rails logger) writing to the same file? Is there any?

I wondered the same thing.  The tests are passing though, and (correct me if I'm wrong here) they're all effectively going to be synchronous, since each cuke profile gets it's own conjur instance.  

> This is going to cause the audit records to be be written to development.log anytime there isn't an audit_socket config. This is true in the dev/test environment, but it's also true for the production environment that's used in the published OSS Conjur container image.

This seems like a valid concern...

> If that's not possible, I wonder if it would be better to have this write to a file audit.log, and we would update our cukes to check there instead of the server log.

Yeah, I think this is cleaner.   The problem (and this was why I didn't separate them) is that forcing end-users to have to check two different log files for different types of events seems unacceptable.  What we want i (I think) is a way for this to affect only dev and ci.